src/main/kotlin/no/fdk/concept_catalog/model/SortField.kt

```diff
@@ -0,0 +1,18 @@
+package no.fdk.concept_catalog.model
+
+import org.springframework.data.domain.Sort
+
+enum class SortFieldEnum(val label: String) {
+    SIST_ENDRET("sistEndret"),
+    ANBEFALT_TERM("anbefaltTerm"),
```

Comment:
```suggestion
    ANBEFALT_TERM_NB("anbefaltTermNB"),
```

Reply:
Så er det mer forsåelig hvis senere utvider til å støtte sortering på de andre språkene

Reply:
Blir kanskje bedre med dette:
```suggestion
    ANBEFALT_TERM_NB("anbefaltTerm.nb"),
```